scripts/lightweight_curator.py

```diff
@@ -158,7 +167,7 @@ def main():
     # Pass elasticsearch connect arguments.
     es = es_connect_args(elasticsearch_host)
 
-    log(log_info, "Removing indices which are above {percentage} threshold from host '{host}'".format(
+    log(log_info, "Searching for indices which are above {percentage}% threshold from host '{host}'".format(
```

Comment:
This states you are searching for indices which themselves are individually above threshold. 

Maybe say something like "Removing/Searching for oldest indices to remove first when total storage exceeds threshold".

Reply:
I took inspiration from your suggestion.
Could you please check if it looks good now?